pkg/apis/core/validation/validation.go

```diff
@@ -3892,12 +3911,18 @@ func validateHostIPs(pod *core.Pod) field.ErrorList {
 func ValidatePodSpec(spec *core.PodSpec, podMeta *metav1.ObjectMeta, fldPath *field.Path, opts PodValidationOptions) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	var gracePeriod int64
+	if spec.TerminationGracePeriodSeconds != nil {
+		//this could happen in tests
+		gracePeriod = *spec.TerminationGracePeriodSeconds
```

Comment:
Apparently we don't validate this field (!!!!), which would have made all previous tests fail.  I'll allow it.